src/operator/controllers/kafkaserverconfig_controller.go

```diff
@@ -258,10 +271,46 @@ func (r *KafkaServerConfigReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, err
 	}
 
+	if err := r.uploadKafkaServerConfig(ctx, kafkaServerConfig); err != nil {
+		logrus.WithError(err).Error("failed to upload KafkaServerConfig to cloud")
+		return ctrl.Result{RequeueAfter: time.Minute}, err
+	}
+
 	r.RecordNormalEvent(kafkaServerConfig, ReasonAppliedKafkaServerConfigFailed, "successfully applied server config")
 	return ctrl.Result{}, nil
 }
 
+func (r *KafkaServerConfigReconciler) uploadKafkaServerConfig(ctx context.Context, kafkaServerConfig *otterizev1alpha1.KafkaServerConfig) error {
+	if r.otterizeClient == nil {
+		return nil
+	}
+
+	err := r.otterizeClient.ReportKubernetesNamespace(ctx, kafkaServerConfig.Namespace)
```

Comment:
I believe the other place that reports namespaces uses a cache to avoid reporting twice, see if you can use it too (maybe put it in the client?)

Reply:
I don't want to cache here, since it's very rare to create this object, if it's reported twice it due to removal and re-apply of the resource. So we do want to report it and log the change in the access graph